config/default_configs/default_config.yml

```diff
@@ -31,11 +31,11 @@ z_max:
   help: "Model top height. Default: 30km"
   value: 30000.0
 kappa_4_vorticity:
-  help: "Hyperdiffusion parameter for vorticity"
-  value: 2.0e17
+  help: "Hyperdiffusion coefficient for vorticity"
```

Comment:
Can you please add a docstring with the units? (They are slightly strange, m/s, because of the third power in length in the hyperdiffusion.)